.github/workflows/run-network-tests.yml

```diff
@@ -189,39 +189,115 @@ jobs:
           export NO_STORAGE=${{ matrix.no_storage }}
           tests/network-tests/run-tests.sh ${{ matrix.scenario }}
 
+  build_master_testing:
+    name: Build joystream node images for mainnet-testing
+    runs-on: ubuntu-latest
+    outputs:
+      use_artifact: ${{ steps.compute_shasum.outputs.shasum }}-joystream-node-docker-image.tar.gz
+    steps:
+      - name: check out master repo
+        uses: actions/checkout@v3
+        with:
+          repository: Joystream/joystream
+          ref: master
+
+      - id: compute_shasum
+        env:
+          RUNTIME_PROFILE: 'TESTING'
+        run: |
+          export RUNTIME_CODE_SHASUM=`scripts/runtime-code-shasum.sh`
+          echo "::set-output name=shasum::${RUNTIME_CODE_SHASUM}"
+
+      - name: Setup cache directory
```

Comment:
I think we should be able to simply just fetch the image from docker hub because the `joystream-node-docker-dev.yml` workflow pushes all the profile builds when code is committed to a branch.